picture-stealer/src/main/java/com/bobocode/picturestealer/service/CameraService.java

```diff
@@ -0,0 +1,44 @@
+package com.bobocode.picturestealer.service;
+
+import com.bobocode.picturestealer.dto.Camera;
+import com.bobocode.picturestealer.entity.CameraEntity;
+import com.bobocode.picturestealer.manager.CameraRepository;
+import jakarta.persistence.LockModeType;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.jpa.repository.Lock;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @author "Maksym Oliinyk"
+ */
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class CameraService {
+
+    private final CameraRepository cameraRepository;
+
+    @Lock(LockModeType.OPTIMISTIC_FORCE_INCREMENT)
+    @Transactional(propagation = Propagation.REQUIRED)
```

Comment:
@WHALE88 transaction propagation is `REQUIRED` by default. So you can just put `@Transactional`

Reply:
@tboychuk agree, but I used it cause I find its visual more satisfying